packages/react-components/style/output-components-style.css

```diff
@@ -93,27 +97,80 @@ canvas {
     align-self: center;
 }
 
-.table-tree>tbody>tr:nth-child(1)>th {
-    background-color: var(--theia-editor-background);
-    position: sticky;
-    top: 0;
+.table-tree {
+    display: grid;
+    border-collapse: collapse;
+    border-left: 1px solid #CCC;
+}
+
+.table-tree thead, .table-tree tbody, .table-tree tr {
+    display: contents;
+}
+
+.table-tree th {
+    position: relative;
+    padding: 3px 0px 3px 5px;
+    background-color: var(--theia-list-hoverBackground) !important;
```

Comment:
It would be good to match the header background color of the events table. However, that's not straight forward.  Probably it should have a separate theming task to handle themes better in all the views.